### PR TITLE
Update probe-rs mentions

### PR DESCRIPTION
**Pull request by bugadani**
This PR modernizes the displayed command suggestions in hints and settings. I've also fixed a few typos and clarified which setting overrides debuggerExecutable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -334,7 +334,7 @@
                         "properties": {
                             "server": {
                                 "type": "string",
-                                "description": "Optionally onnect to an existing `probe-rs-debugger` session on IP and Port, e.g. '127.0.0.1:50000'",
+                                "description": "Optionally onnect to an existing `probe-rs dap-server` session on IP and Port, e.g. '127.0.0.1:50000'",
```
Comment (noppej) [inline package.json]: Missed a typo with 'onnect' :)

**Review by noppej — APPROVED** [review]
Thanks for doing this. I found one more typo, but other than that, I am good with this.

FYI. You need to manually bump the version number to 0.21.3 if you want this PR to push a new version to the VSCode marketplace.

**Review by noppej — CHANGES_REQUESTED** [review]
Let's hold merging this until I can look at why the CI is broken.

**Review by Yatekii — APPROVED** [review]
Thanks!

**Comment by noppej** [comment]
@bugadani CI is now fixed. Thanks for helping to cleanup the docs/usability issues!

**Review by noppej — APPROVED** [review]

**Comment by bugadani** [comment]
My pleasure :)

**Pull request merged**